modules/apps/object/object-web/src/main/java/com/liferay/object/web/internal/object/definitions/display/context/ObjectDefinitionsActionsDisplayContext.java

```diff
@@ -59,13 +60,15 @@ public ObjectDefinitionsActionsDisplayContext(
 		ObjectActionTriggerRegistry objectActionTriggerRegistry,
 		ModelResourcePermission<ObjectDefinition>
 			objectDefinitionModelResourcePermission,
-		JSONFactory jsonFactory) {
+		JSONFactory jsonFactory,
```

Comment:
I believe we can order the parameters alphabetically. Could you do that?
 It was wrong before your new code.